packages/bsky/src/config.ts

```diff
@@ -356,6 +382,34 @@ export class ServerConfig {
   get maxThreadDepth() {
     return this.cfg.maxThreadDepth
   }
+
+  get disableSsrfProtection(): boolean {
+    return this.cfg.disableSsrfProtection ?? false
+  }
+
+  get proxyAllowHTTP2(): boolean {
+    return this.cfg.proxyAllowHTTP2 ?? false
+  }
+
+  get proxyHeadersTimeout(): number {
+    return this.cfg.proxyHeadersTimeout ?? 10e3
+  }
```

Comment:
Just a headsup that this change in timeout was disruptive at least for the PDS use case.  Maybe we should set both defaults to something less conservative, like 30s.

Reply:
As it stands, this will only impact `getBlob` calls from the "blob-resolver". The current timeout there is actually 5 seconds (see bellow).

https://github.com/bluesky-social/atproto/blob/c72145dbeb2d67068bc28c00a13447e0d382d121/packages/bsky/src/api/blob-resolver.ts#L133C1-L133C59

Should I:
1) Lower the timeout to match current behavior ?
2) Increase to timeout to future proof (when these config become used for other PDS proxying purposes) ?
3) Leave it like that?